src/01-function-calling/example.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Sample code from https://platform.openai.com/docs/api-reference/chat/create, lightly TS-ified
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+
+import dotenv from "dotenv"
+import OpenAI from "openai"
+
+dotenv.config()
+const openai = new OpenAI()
+
+async function main() {
+  const messages: OpenAI.Chat.Completions.ChatCompletionMessageParam[] = [
+    { role: "user", content: "What's the weather like in Boston today?" },
+  ]
+
+  const tools: OpenAI.Chat.Completions.ChatCompletionTool[] = [
+    {
+      type: "function",
+      function: {
+        name: "get_current_weather",
+        description: "Get the current weather in a given location",
+        parameters: {
+          type: "object",
+          properties: {
+            location: {
+              type: "string",
+              description: "The city and state, e.g. San Francisco, CA",
+            },
+            unit: { type: "string", enum: ["celsius", "fahrenheit"] },
+          },
+          required: ["location"],
+        },
+      },
+    },
+  ]
+
+  const response = await openai.chat.completions.create({
+    model: "gpt-3.5-turbo",
```

Comment:
I can look into this more if you don't know off hand, but would the above function work with gpt-3.5, which I don't believe supports browsing? It's not clear to me how these functions would retrieve real time information without it.

Reply:
Ah yeah, this was just sample code straight from their docs site. I don't think it even includes the definition of `get_current_weather` which _would_ be needed for this to do anything. 

Probably clearer if I just delete this file from the repo.